lib/radiator_web/controllers/api/person_controller.ex

```diff
@@ -0,0 +1,59 @@
+defmodule RadiatorWeb.Api.PersonController do
+  use RadiatorWeb, :rest_controller
+
+  alias Radiator.Directory.Editor
+  alias Radiator.Directory.Network
+
+  require Logger
+
+  # TODO: move this into :rest_controller for all rest controllers
+  def action(conn, _) do
```

Comment:
what does this do?

Reply:
add the user to the action function calls as param, vs. having to fetch it in every action with current_user. And as all our REST api needs a user it seems appropriate. It is also a style used in the Phoenix book.